ee/codegen/src/__test__/__snapshots__/inputs.test.ts.snap

```diff
@@ -60,8 +60,8 @@ exports[`Inputs > write > should generate correct code when Inputs has variables
 
 
 class Inputs(BaseInputs):
-    var_1: str
-    var_2: str
+    input_: str
```

Comment:
[q] is this intentional naming?

Reply:
Yeah – I wanted to make the default name "input", but that's a reserved keyword in python, so a common approach is to add a trailing "_"

Maybe `Inputs.input_` is bad though. Open to alternative naming suggestion, but I had a slight preference for this over `var`

Reply:
Ah I see. Yup I like input_ over the other one as well